picard/tagger.py

```diff
@@ -1071,7 +1103,8 @@ def process_picard_args():
                         help="display version information and exit")
     parser.add_argument("-V", "--long-version", action='store_true',
                         help="display long version information and exit")
-    parser.add_argument('FILE', nargs='*')
+    parser.add_argument('FILE_OR_URL', nargs='*',
+                        help="the file(s), URL(s) and MBID(s) to load")
```

Comment:
Should we include a brief description of the formats of the inputs? Most people will probably understand the files and URLs, but not the required format for MBIDs.

Reply:
I don’t think so. The help messages should be short and substantive. It should be documented in some place (my suggestion is picard docs and maybe the official wiki article about MBID) but not in --help imo.